src/translations/en.json

```diff
@@ -3350,7 +3350,7 @@
                 },
                 "trigger": {
                   "label": "Triggered by",
-                  "no_triggers": "No triggers have an ID set. You can set an ID using the trigger menu button.",
+                  "no_triggers": "No triggers have an ID set. You can edit IDs using each trigger's menu.",
```

Comment:
Do we have a user term for the 3 dot menu?

@googanhiem

Reply:
Checked with others, Docs call them a "three-dot menu".

`"No triggers have an ID set. Use the three-dot menu on the trigger and select Edit ID to assign one."`

Reply:
OK, I've changed to the suggested string, only used "a trigger", not "the trigger" as it's up to the user to pick one or more to assign an ID at this point.

Reply:
We do have a handful of references to "Overflow menu" in the current strings.